dronegen/apt.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import "path"
+
+// This function calls the build-apt-repos tool which handles the APT portion of RFD 0058.
+func promoteAptPipeline() pipeline {
+	return getAptPipelineBuilder().buildPromoteOsPackagePipeline()
+}
+
+func migrateAptPipeline(triggerBranch string, migrationVersions []string) pipeline {
+	return getAptPipelineBuilder().buildMigrateOsPackagePipeline(triggerBranch, migrationVersions)
+}
+
+func getAptPipelineBuilder() *OsPackageToolPipelineBuilder {
+	optpb := NewOsPackageToolPipelineBuilder(
+		"drone-s3-aptrepo-pvc",
+		"golang:1.18.1-bullseye",
```

Comment:
Any reason Go 18.1 is used for apt and 18.3 is used for rpm?

Reply:
When I wrote the APT portion only 18.1 was available. Updated both to 18.4